src/Ledger/Utxo/Properties.lagda

```diff
@@ -8,24 +8,25 @@ open import Ledger.Transaction
 
 module Ledger.Utxo.Properties (txs : TransactionStructure) where
 
-open import Prelude
-open import Ledger.Prelude
+open import Prelude hiding (_+_)
+open import Ledger.Prelude renaming (_+_ to _+ℕ_)
+
+open import Algebra              using     (CommutativeMonoid ; Monoid)
+open import Algebra.Morphism     using     (module MonoidMorphisms ; IsMagmaHomomorphism)
+open import Data.Integer         using     (ℤ; _⊖_ ; _-_ ; 0ℤ)
+                                 renaming  (_+_ to _+ℤ_ ; +_ to _⁺ ; -_ to _⁻)
+open import Data.Integer.Ext     using     (posPart ; negPart)
+open import Data.Nat.Properties  hiding    (_≟_)
+open import Data.Sign            using     (Sign)
+open import Relation.Binary      using     (IsEquivalence)
 
-import Data.List as List
-import Data.Nat as ℕ
-open import Algebra.Morphism
-open import Data.Nat.Properties using
-  (+-0-commutativeMonoid; +-0-monoid; +-comm; +-identityʳ; +-assoc; +-∸-assoc; m≤n⇒m≤n+o; ≤″⇒≤; m+[n∸m]≡n)
-open import Data.Sign using (Sign)
-open import Data.Integer as ℤ using (ℤ; _⊖_)
-open import Data.Integer.Ext
-import Data.Integer.Properties as ℤ
-open import Interface.ComputationalRelation
-open import Relation.Binary
 open import Tactic.Cong
 open import Tactic.EquationalReasoning
 open import Tactic.MonoidSolver
 
+import Data.Nat as ℕ
+import Data.Integer.Properties as ℤ
```

Comment:
Minor remarks on some of the existing code:

1. Referring to `Data.Nat` as `ℕ` is okay, I suppose... but doesn't this mean the natural numbers as called `ℕ.ℕ`?
2. Referring to the `Data.Integer.Properties` module as `ℤ` seems even stranger still.

(I tried to stick with conventions such as these wherever possible.)